AmpegManualTestProject.txt

```diff
@@ -0,0 +1,104 @@
+--Ampeg Manual Test Project--
+https://ampeg.com/
+
+1. You decide what should be priority (to test) in the webapp.
+2. 6 manual test cases total.
+3. Ensure to include what 'should' be included in a manual test (title, summary, etc).
+4. Ensure that 5 tests consist of navigating to more than one page.
+5. Ensure that 2 tests consist of navigating to 3 or more pages.
+
+1:
+
+Title: Find Contact Info
+
+Summary: The user will navigate the website and discover the company's contact info.

```

Comment:
I would reword this to describe what the user is testing.

E.G:
Title: Company contact info
Summary: Verifies that company contact information is displayed